lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_get_udids.rb

```diff
@@ -0,0 +1,93 @@
+require 'fastlane/action'
+require 'open3'
+require 'shellwords'
+require 'googleauth'
+require_relative '../helper/firebase_app_distribution_helper'
+require_relative '../helper/firebase_app_distribution_error_message'
+require_relative '../client/firebase_app_distribution_api_client'
+require_relative '../helper/firebase_app_distribution_auth_client'
+
+module Fastlane
+  module Actions
+    class FirebaseAppDistributionGetUdidsAction < Action
+      extend Auth::FirebaseAppDistributionAuthClient
+      extend Helper::FirebaseAppDistributionHelper
+
+      def self.run(params)
+        auth_token = fetch_auth_token(params[:service_credentials_file], params[:firebase_cli_token])
+        fad_api_client = Client::FirebaseAppDistributionApiClient.new(auth_token, params[:debug])
+
+        app_id = params[:app]
```

Comment:
Consider adding support for getting the `app_id` from the plist file like we do in the main action

https://github.com/fastlane/fastlane-plugin-firebase_app_distribution/blob/c6bce4c719213bc07162ceaba4685ffd1874fe57/lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb#L80

Reply:
Gotcha that makes sense